jupyterlab/commands.py

```diff
@@ -1769,7 +1769,7 @@ def _fetch_package_metadata(registry, name, logger):
         logger.debug('Fetching URL: %s' % (req.get_full_url()))
     try:
         with contextlib.closing(urlopen(req)) as response:
-            return json.loads(response.decode('utf-8'))
+            return json.loads(response.read().decode('utf-8'))
```

Comment:
you could just use `json.load`, right? That's simpler.

Reply:
never mind, that was the original code. Okay, this looks good.

Reply:
Can you be more explicit? It originally was `json.load(response)`, but the response was in bytes and needed to be decoded.